gradle.properties

```diff
@@ -41,3 +41,5 @@ pythonVersion = 3.8
 
 # skipDockerTests is used to skip the tests that require Docker to be running.
 skipDockerTests = true
+# The default backend for the JDBC tests, you set it to h2 or mysql
+jdbcBackend = h2
```

Comment:
I think we should figure out a better way for test between h2 and MySQL.

1) For UT, we can use H2.
2) For IT, we'd better run both.

So, we don't have to make a configuration there, WDYT?

Reply:
OK, Add the configuration here is indeed unnecessary as `H2` is the default one. 

> For UT, we can use H2.
For IT, we'd better run both.

Yeah, the strategy is correct. I will check it again.